docs/docusaurus/docs/core/trigger_actions_based_on_results/create_a_checkpoint_with_actions.md

```diff
@@ -11,14 +11,14 @@ import PrereqValidationDefinition from '../_core_components/prerequisites/_valid
 
 A Checkpoint executes one or more Validation Definitions and then performs a set of Actions based on the Validation Results each Validation Definition returns.
 
-<h2>Prerequisites</h2>
+## Prerequisites
 
 - <PrereqPythonInstalled/>.
 - <PrereqGxInstalled/>.
 - <PrereqPreconfiguredDataContext/>. In this guide the variable `context` is assumed to contain your Data Context.
 - <PrereqValidationDefinition/>.
 
-### Procedure
+## Procedure
```

Comment:
**Note for reviewer:** apologies for scope creep but I noticed some header level issues on this page when I copied it to scaffold the new page, so I went ahead and fixed the problems on this page.